internal/services/post_service.go

```diff
@@ -91,3 +92,12 @@ func (service *PostService) CreatePost(req *models.PostCreateRequestDTO, usernam
 
 	return &postDto, nil, http.StatusCreated
 }
+
+func (service *PostService) FindPostsByUser(username string, offset, limit int) ([]models.Post, error) {
+	posts, err := service.postRepo.FindPostsByUser(username, offset, limit)
+	if err != nil {
+		return nil, err
+	}
+
+	return posts, nil
```

Comment:
Das ist zu einfach. Es wurden nicht alle Fehlerfälle betrachtet, die in Postman zu finden sind.
- User Not Found
- Unauthorized im Controller
Außerdem musst du DTOs wie in Postman definiert zurückgeben.